qemu/tests/rh_kernel_update.py

```diff
@@ -185,39 +109,12 @@ def is_virtio_driver_installed():
 
         for driver in virtio_drivers:
             if driver not in o:
-                logging.debug("%s has not been installed." % driver)
+                logging.debug("%s has not been installed.", driver)
                 return False
-            logging.debug("%s has already been installed." % driver)
+            logging.debug("%s has already been installed.", driver)
```

Comment:
For RHEL8,  from kernel 4.17,  `VIRTIO` ,  `VIRTIO_PCI `  and `VIRTIO_RING`  are configured as build-in driver, can not get them from lsmod.
```
CONFIG_RING_BUFFER=y
CONFIG_VIRTIO=y
CONFIG_VIRTIO_PCI=y
```

I think we can remove the step of checking "is_virtio_driver_installed".  Linux vm can load virtio driver automatically when needed.

Reply:
ok, thanks for the information.